mlflow/server/js/src/experiment-tracking/components/experiment-page/components/header/ExperimentViewHeader.tsx

```diff
@@ -1,8 +1,13 @@
-import React, { useMemo } from 'react';
+import React, { useMemo, useState } from 'react';
```

Comment:
Let's remove unused functions/components

Reply:
@hubertzub-db Is it possible to turn this warning into an error?

Reply:
@harupy yeah, let's do this https://github.com/databricks/universe/pull/264618

Reply:
Thanks @hubertzub-db ! I'll let this take effect on the next sync :)

Reply:
@dbczumar Let's disable the problem matcher by running the following command after `setup-node` until the next sync.

```
echo "::remove-matcher owner=eslint-compact::"
```